xonsh/readline_shell.py

```diff
@@ -232,9 +235,24 @@ def parseline(self, line):
         """Overridden to no-op."""
         return '', line, line
 
+    def _rebind_case_sensitive_completions(self):
```

Comment:
This doesn't use self, so I think think that this can safely be a method at global scope.